backend/src/common/mail-service/mail-service.package.ts

```diff
@@ -11,7 +11,7 @@ class MailService implements IMailService {
         const transportOptions: Options = {
             host: process.env.SMTP_HOST,
             port: Number(process.env.SMTP_PORT),
-            secure: true,
+            secure: false,
```

Comment:
Our coaches wanted it to be true, as I remember

Reply:
You are correct, I didn't mean to push it.